documentation/assemblies/assembly-getting-started-strimzi-admin.adoc

```diff
@@ -6,8 +6,13 @@
 = Strimzi Administrators
 
 {ProductName} includes several custom resources.
-By default, permission to create, edit, and delete these resources is limited to Kubernetes cluster administrators.
-If you want to allow non-cluster administators to manage {ProductName} resources, you must assign them the Strimzi Administrator role.
+By default, permission to view, create, edit, and delete these resources is limited to Kubernetes cluster administrators.
+{ProductName} provides two additional `ClusterRoles` which you can use to assign these rights also to other users.
+The role `strimzi-view` allows used to view and list the {ProductName} resources.
+The role `strimzi-admin` allows them to also create, edit or delete them.
```

Comment:
"them" is used twice here with different meanings, maybe it's better to specify "users" and "{ProductName} resources" ?